packages/firebase_database/lib/src/database_reference.dart

```diff
@@ -125,10 +128,65 @@ class DatabaseReference extends Query {
   ///
   /// remove() is equivalent to calling set(null)
   Future<Null> remove() => set(null);
+
+  /// Determine the next transaction key to be used. 0 is used if there are
```

Comment:
No need for dartdoc comments for private methods

Reply:
Removed.